README.md

```diff
@@ -48,7 +48,7 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
 | BD                                                                                    | San Diego, CA | **🔒 Closed 🔒** Data Science                                                                                     |
 | [Bridgewater Associates](https://boards.greenhouse.io/bridgewater89/jobs/5830071002)                                                                                         | Westport, CT                         |      Investment Engineer Intern |
 | [Bristol Myers Squibb](https://careers.bms.com/jobs/R1546751?lang=en-us)                                                                                         | Tampa, FL                         |  Data Science
-| [Capital One](https://www.capitalonecareers.com/job/mclean/technology-internship-program-summer-2023/31238/31914110656)    | Multiple Locations in United States | Software Engineering (no visa sponsorship; instant rejection if you declare sponsorship need in the application), Machine Learning, Cyber Security 							|
+| [Capital One](https://www.capitalonecareers.com/job/lean/technology-internship-program-summer-2023/31238/31914110656)    | Multiple Locations in United States | Software Engineering (no visa sponsorship; instant rejection if you declare sponsorship need in the application), Machine Learning, Cyber Security 							|
```

Comment:
why did you make this change?

Reply:
sorry, that was unintended. I must've accidentally clicked something

Reply:
I have resolved the error by editing it.